server/src/v2/modules/db.js

```diff
@@ -116,59 +116,51 @@ function initialise() {
 
 initialise();
 
-export function db_addSecret(secretObject) {
+function getStatement(preparedStatement, statementParams) {
+  //Returns data
+  let row;
   try {
-    databaseFile.prepare(INSERT_SECRET_QUERY).run(secretObject.secret_id,secretObject.secret_text, secretObject.passphrase, secretObject.expiry_date, secretObject.method);
+    row = databaseFile.prepare(preparedStatement).get(statementParams);
   }
   catch (err) {
     return { data: null, code: 500, human_code: `failure, ${err}` };
   }
-  return { data: null, code: 200, human_readable_code: "Success" };
+  return { data: row, code: 200, human_readable_code: "Success" };
```

Comment:
I think that these codes are still a little unnecessary - they aren't used anywhere in the application, and it can never directly accessed by a HTTP request.

When success it should return `{data: row}`.
When errored it should return `{data: false, error: err}` (or null)

Reply:
The code seems to be used in `routes/secret/get.js`, which passes them back out in the event of an error. The only issue that I can see with removing the codes is that the code there reads the status, and may behave unpredictably if those status codes are removed.